clang/lib/StaticAnalyzer/Checkers/ArrayBoundCheckerV2.cpp

```diff
@@ -83,6 +83,8 @@ class StateUpdateReporter {
     AssumedUpperBound = UpperBoundVal;
   }
 
+  bool assumedNonNegative() { return AssumedNonNegative; }
```

Comment:
I'd prefer to keep the current name because it's more natural if we're looking at this from the outside (the get/has variants are awkward because `AssumedNonNegative` is not a noun phrase); and it's just an implementation detail that this is a getter for a private data member.